Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -69,74 +119,59 @@ function maybe_prompt_user() {
     fi
 }
 
-
-OPTIND=1  # Reset in case getopts has been used previously in the shell.
-while getopts "y" opt; do
-    case "$opt" in
-        \?)
-            exit 1
-            ;;
-        y)  ASSUME_YES=true
-            ;;
-        q)  QUIET=true
-            ;;
-    esac
-done
-
-APT_GET="sudo apt-get"
-if $ASSUME_YES; then
-    APT_GET="$APT_GET --assume-yes"
-fi
-if $QUIET; then
-    APT_GET="$APT_GET -qq"
-fi
-
-if ! dpkg-query -l "lsb-release"; then
-    $APT_GET install lsb-release
-fi
 # possibly grab a newer cmake for older ubuntu releases
-read -r UBUNTU_CODENAME <<<$(lsb_release -c -s)
-if [ "$UBUNTU_CODENAME" = "precise" ]; then
+if [ ${RELEASE_CODENAME} == "precise" ]; then
     sudo add-apt-repository ppa:george-edison55/precise-backports -y
-elif [ "$UBUNTU_CODENAME" = "trusty" ]; then
+elif [ ${RELEASE_CODENAME} == "trusty" ]; then
     sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 fi
 
+$APT_GET update
+
+echo "$sep"
+echo "Add use to dialout group to allow managing serial ports"
+echo "$sep"
 sudo usermod -a -G dialout $USER
+echo "Done!"
 
+echo "$sep"
+echo "Removing modemmanager that could conflict with firmware uploading"
+echo "$sep"
 if dpkg-query -l "modemmanager"; then
     $APT_GET remove modemmanager
 fi
-$APT_GET update
-
-if apt-cache search python-wxgtk3.0 | grep wx; then
-    SITL_PKGS+=" python-wxgtk3.0 libtool-bin"
-else
-    # we only support back to trusty:
-    SITL_PKGS+=" python-wxgtk2.8"
+echo "Done!"
+
+# Check for graphical package for MAVProxy
+if [[ ! -v SKIP_AP_GRAPHIC_ENV ]]; then
```

Comment:
Please don't use "-v" like this; someone playing with your dockerfile may try setting one of the vars to 0 - esp. in the format you're using.

Please just use truthiness values.

Reply:
done